src/iuse.cpp

```diff
@@ -9187,6 +9187,35 @@ int iuse::disassemble( player *p, item *it, bool, const tripoint & )
     return 0;
 }
 
+int iuse::gobag( player *p, item *it, bool, const tripoint & )
+{
+    auto items = item_group::items_from( "gobag_contents", calendar::turn );
+    bool content_fits = it->typeId() == "personal_gobag";
```

Comment:
This bool isn't checked anywhere except for `for( auto &content : items ) {` cycle below, so it's best to declare it inside the cycle.

Reply:
It needs to be declared and initialized outside since it's possible to use it without altering it in the loop. The value needs to be remember between loops too. Moving it inside the loop will cause the shirts/socks to not match in size.